### ISSUE TITLE
Xdebug Missing Continuous Commands

### ISSUE THREAD

**Issue by x558li** [martomo/SublimeTextXdebug#196]
I added a breakpoint in a file. When I start debugging and open the browser, the Xdebug panels shows up correctly, it stops at the first breakpoint with context shown, the server hangs. However, it doesn't open up the window to show current running line and the continuation commands are missing, which means I cannot run to next breakpoint or step to the next line. I also tried the short key Ctrl+Shift+F5, not working. But, i have Evaluate, Execute, Status. After I stop debugging, the website is loaded correctly. 
![Xdebug](https://user-images.githubusercontent.com/35274349/55427190-77617400-5554-11e9-9529-291f2f9ce865.png)

### Sublime Text
__Operating system:__

Ubuntu 16.04.6 LTS (Xenial Xerus)
__Installed version/build:__

Sublime Text Build 3200
__Python version:__

Python 2.7.12

### Server

__PHP/Xdebug version:__

PHP 7.3.3-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Mar  7 2019 20:31:26) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.3, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.3-1+ubuntu16.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.7.0, Copyright (c) 2002-2019, by Derick Rethans

## Configuration

__php.ini/xdebug.ini__

```ini
[xdebug]
# ...
zend_extension=/usr/lib/php/20180731/xdebug.so
xdebug.remote_autostart = 1
xdebug.remote_enable = 1
xdebug.remote_handler = dbgp
xdebug.remote_host = localhost
xdebug.remote_log = /tmp/xdebug.log
xdebug.remote_mode = req
xdebug.remote_port = 9000
```

__Packages/User/Xdebug.sublime-settings__

```json
{
    "path_mapping": {
    },
    "url": "http://localhost/subscription/",
    "ide_key": "sublime.xdebug",
    "host": "localhost",
    "port": 9000,
    "max_children": 32,
    "max_data": 1024,
    "max_depth": 3,
    "break_on_start": false,
    "break_on_exception": [
        "Fatal error",
        "Catchable fatal error",
        "Warning",
        "Parse error",
        "Notice",
        "Strict standards",
        "Deprecated",
        "Xdebug",
        "Unknown error"
    ],
    "close_on_stop": false,
    "super_globals": true,
    "fullname_property": true,
    "hide_password": false,
    "pretty_output": false,
    "launch_browser": true,
    "browser_no_execute": false,
    "disable_layout": false,
    "debug_layout" : {
        "cols": [0.0, 0.5, 1.0],
        "rows": [0.0, 0.7, 1.0],
        "cells": [[0, 0, 2, 1], [0, 1, 1, 2], [1, 1, 2, 2]]
    },
    "breakpoint_group": 2,
    "breakpoint_index": 1,
    "context_group": 1,
    "context_index": 0,
    "stack_group": 2,
    "stack_index": 0,
    "watch_group": 1,
    "watch_index": 1,
    "breakpoint_enabled": "circle",
    "breakpoint_disabled": "dot",
    "breakpoint_current": "",
    "current_line": "bookmark",
    "python_path" : "",
    "debug": false
}
```
__*.sublime-project__

```json
{
	"folders":
	[
		{
			"follow_symlinks": true,
            "path": "/var/www/html/subscription"
		}
	],
    "settings": {
        "xdebug": {
             "url": "http://localhost/subscription"
        }
    }
}
```

## Logs

__Console output:__

```
startup, version: 3200 linux x64 channel: stable
executable: /opt/sublime_text/sublime_text
working dir: /
packages path: /home/.config/sublime-text-3/Packages
state path: /home/.config/sublime-text-3/Local
zip path: /opt/sublime_text/Packages
zip path: /home/.config/sublime-text-3/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 0.128594
startup time: 0.337649
first paint time: 0.380693
git: tracking working dir /var/www/html/subscription
reloading plugin Default.arithmetic
...
plugins loaded
git: untracking working dir /var/www/html/subscription
Package Control: Skipping automatic upgrade
```
__Packages/User/Xdebug.log:__

[Xdebug.log](https://github.com/martomo/SublimeTextXdebug/files/3035511/Xdebug.log)
```


```

**Comment by x558li** [comment]
I tried https://github.com/martomo/SublimeTextXdebug/issues/184. After changing protocol.py, the menu showed up but turned grey, so I cannot even start debugging.

**Comment by gepeixoto** [comment]
I've fixed the problem. You can clone from my fork:
https://github.com/gepeixoto/SublimeTextXdebug.git

**Comment by alexkuc** [comment]
Solution provided by @gepeixoto worked for me! Thank you so much! 😄

**Comment by aneetkukreja1** [comment]
Thanks alot!! @gepeixoto I was struggling for hours. It fixed my issue.

**Comment by martomo** [comment]
Closing this issue as this should be resolved with release 1.1.2, in case the issue was not resolved please let me know.
